src/main/java/com/github/sbaudoin/yamllint/Linter.java

```diff
@@ -162,7 +214,20 @@ private Linter() {
      * @param file the file whose content has been passed as the <var>buffer</var>. May be <code>null</code>.
      * @return the list of problems found on the passed YAML string
      */
-    public static List<LintProblem> run(String buffer, YamlLintConfig conf, File file) {
+    public static List<LintProblem> run(String buffer, YamlLintConfig conf, @Nullable File file) {
```

Comment:
One `File file` parameter already used the `@Nullable` annotation, so I simply placed it on all the methods that used that same parameter, both for consistency and to make it clearer that the other parameters are _not_ nullable.

Otherwise I tried not to change the existing code, and tried to mimic the style of the Javadocs.